src/main/java/nl/tudelft/ewi/devhub/server/web/resources/ProjectAssignmentsResource.java

```diff
@@ -414,7 +400,32 @@ public void persistMasteries(
 		);
 
 		deliveries.merge(delivery);
-
 	}
 
+    @Data
+    public static class AutoGradeResult {
+        private List<Long> ids;
+    }
+
+
+    @POST
+    @Transactional
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("{assignmentId}/last-delivery/auto-masteries")
+    public void uploadAutoGradingGrade(@PathParam("assignmentId") long assignmentId, AutoGradeResult result) {
```

Comment:
*Plus you would have the option to make the client side type safe if you shared the data object/interface*